content/source/docs/cloud/api/audit-trails.html.md

```diff
@@ -41,15 +41,16 @@ The audit trails API exposes a stream of audit events, which describe changes to
 | Parameter | Description                                                                                                                                                                      |
 | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `since`   | **Optional.** Returns only audit trails created after this date (UTC and in [ISO8601 Format](https://www.iso.org/iso-8601-date-and-time-format.html) - YYYY-MM-DDTHH:MM:SS.SSSZ) |
-| `page`    | **Optional.** If omitted, the endpoint will return the first page.                                                                                                               |
+`page[number]`      | **Optional.** If omitted, the endpoint will return the first page.
+`page[size]`        | **Optional.** If omitted, the endpoint will return 1000 audit events per page.                                                                                                             |
 
 ### Sample Request
 
 ```shell
 $ curl \
   --header "Authorization: Bearer $TOKEN" \
   --request GET \
-  https://app.terraform.io/api/v2/organization/audit-trail?page=1&since=2020-05-30T17:52:46.000Z
+  https://app.terraform.io/api/v2/organization/audit-trail?page[number]=1&since=2020-05-30T17:52:46.000Z
```

Comment:
Won't this need to be wrapped in quotes or have the `?`, `[`, and `]` encoded?

Reply:
I'll go quotes, bash will eat the & and background, zsh is nice. There's a bit of inconsistency with regards to sample curls and parameters, some have some, some are quoted regardless.